hello.java

```diff
@@ -1,5 +1,5 @@
 public class Hello{
 	public static void public static void main(String[] args) {

```

Comment:
you know i think when i was typing it up the auto fill function may have played a hand.